DatadogWebViewTracking/Sources/WebViewTracking+objc.swift

```diff
@@ -0,0 +1,113 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-Present Datadog, Inc.
+ */
+
+import Foundation
+#if os(tvOS)
+#warning("Datadog WebView Tracking does not support tvOS")
+#else
+import WebKit
+#endif
+
+@objc
```

Comment:
/change-request We aim to hide these interfaces from Swift autocompletion. As stated in the [RFC](https://datadoghq.atlassian.net/wiki/spaces/RUMP/pages/3157787243/RFC+-+Modular+Objective-C+Interface), the interfaces should read:
```objective-c
@objc(DDWebViewTracking) @_spi(objc)
public final class objc_WebViewTracking: NSObject
```

Reply:
Thanks for flagging this 💪! I forgotten that we established this convention. Now everything should be up-to-date, including the PR comment and `+objc` file location ✅.